### PR TITLE
Day 1 of Midnight Sun spoiler season!

### PR DESCRIPTION
**Pull request by plural**
day 1 of spoiler season's cards.

### CODE REVIEW

File: pack/ms.json [hunk 1]
```diff
@@ -1,4 +1,104 @@
...
+    "type_code": "identity",
+    "uniqueness": false
+  },
+  {
+    "code": "33002",
+    "cost": 3,
+    "deck_limit": 3,
+    "faction_code": "anarch",
+    "faction_cost": 4,
+    "flavor": "We're all alike down here Androids are friends we salute Give us all a f***ing break Ol' Jack is the s*** on my boot.",
```
Comment (distributive) [inline pack/ms.json:30]:
```suggestion
    "flavor": "We're all alike down here\nAndroids are friends we salute\nGive us all a f***ing break\nOl' Jack is the s*** on my boot.",
```
I think we tend to put newlines in the flavour text.

File: v2/printings/midnight_sun.json [hunk 2]
```diff
@@ -1,4 +1,63 @@
...
+    "position": 1,
+    "printed_is_unique": false,
+    "printed_text": "The first time each turn you suffer core damage, you may draw 1 card and sabotage 2. <em>(The Corp trashes 2 cards of their choice from HQ and/or the top of R&D.)</em>",
+    "quantity": 1,
+    "stripped_printed_text": "The first time each turn you suffer core damage, you may draw 1 card and sabotage 2. (The Corp trashes 2 cards of their choice from HQ and/or the top of R&D.)"
+  },
+  {
+    "card_id": "chastushka",
+    "card_set_id": "midnight_sun",
+    "flavor": "We're all alike down here Androids are friends we salute Give us all a f***ing break Ol' Jack is the s*** on my boot.",
```
Comment (distributive) [inline v2/printings/midnight_sun.json:17]:
```suggestion
    "flavor": "We're all alike down here\nAndroids are friends we salute\nGive us all a f***ing break\nOl' Jack is the s*** on my boot.",
```

**Review by distributive — APPROVED** [review]
Is putting the formatting in flavor text something we want to do? Either way this looks good to go up!

File: v2/printings/midnight_sun.json (hunk 2 above)
Comment (plural) [reply v2/printings/midnight_sun.json:17]: yep, thanks.  I'll tidy up the sheet -> JSON converter to make sure this stays.  the sheet definitely has the newlines.

**Pull request merged**